src/request/sign-swap/SignSwapApi.js

```diff
@@ -95,6 +96,13 @@ class SignSwapApi extends PolygonRequestParserMixin(BitcoinRequestParserMixin(To
                 fee: this.parsePositiveInteger(request.fund.fee, true, 'fund.fee'),
                 bankLabel: this.parseLabel(request.fund.bankLabel, true, 'fund.bankLabel'),
             };
+        } else if (request.fund.type === 'CRC') {
+            parsedRequest.fund = {
+                type: 'CRC',
+                amount: this.parsePositiveInteger(request.fund.amount, false, 'fund.amount'),
+                fee: this.parsePositiveInteger(request.fund.fee, true, 'fund.fee'),
+                senderLabel: this.parseLabel(request.fund.senderLabel, true, 'fund.recipientLabel'),
+            };
```

Comment:
Can be reasonably well be merged with the `request.fund.type === 'EUR'` case, I think.